packages/admin/admin/src/FinalForm.tsx

```diff
@@ -251,3 +258,21 @@ export function FinalForm<FormValues = AnyObject>(props: IProps<FormValues>) {
             );
     }
 }
+
+const waitForFormValidationToBeFinished = (form: FormApi<any>): Promise<boolean> => {
```

Comment:
I'd call this waitForValidationToFinish. It is already clear that we're in a form

Reply:
Done https://github.com/vivid-planet/comet/pull/1414/commits/675e9aad5e2616d6753dc82e6034cecfa2c8f186